.github/workflows/test_and_release.yml

```diff
@@ -14,9 +14,9 @@ jobs:
         python-version: ['3.6', '3.7', '3.8']
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
Tangential question: Is there a way to just `actions/checkout@LATEST`, so that we don't have to do these updates?

Yeah, I realize that the API for the action _could_ break slightly with each major version, but it normally doesn't …

Reply:
While `@master` or `@main` will use the tip of the development branch, depending on its name, I don't know of a way to use the latest tagged release. All GitHub projects I know of use specific major versions. It looked like good practice to me, so I did not try to find an alternative.

The GitHub actions have grown rather stable. I don't expect dependabot to create pull requests more than once every few years, so I wouldn't worry about it.